docs/source/java/ipc.rst

```diff
@@ -54,6 +54,15 @@ Now, we can begin writing a stream containing some number of these batches. For
     ArrowStreamWriter writer = new ArrowStreamWriter(root, /*DictionaryProvider=*/null, Channels.newChannel(out));
 
 
+Buffer level compression is also supported by passing a codec:
+Lz4CompressionCodec / ZstdCompressionCodec:
```

Comment:
Looks like there is a missing sentence here? Are these the supported codecs? Something else?